src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts

```diff
@@ -133,22 +133,11 @@ Registry.as<IViewsRegistry>(ViewContainerExtensions.ViewsRegistry).registerViews
 // Register actions
 registerTerminalActions();
 
-const enum Constants {
+export const enum Constants {
 	/** The text representation of `^<letter>` is `'A'.charCodeAt(0) + 1`. */
 	CtrlLetterOffset = 64
 }
 
-// An extra Windows-only ctrl+v keybinding is used for pwsh that sends ctrl+v directly to the
-// shell, this gets handled by PSReadLine which properly handles multi-line pastes. This is
-// disabled in accessibility mode as PowerShell does not run PSReadLine when it detects a screen
-// reader. This works even when clipboard.readText is not supported.
-if (isWindows) {
-	registerSendSequenceKeybinding(String.fromCharCode('V'.charCodeAt(0) - Constants.CtrlLetterOffset), { // ctrl+v
-		when: ContextKeyExpr.and(TerminalContextKeys.focus, ContextKeyExpr.equals(TerminalContextKeyStrings.ShellType, GeneralShellType.PowerShell), CONTEXT_ACCESSIBILITY_MODE_ENABLED.negate()),
```

Comment:
@Tyriar I have removed this and handled the ctrl+v action in "src\vs\workbench\contrib\terminalContrib\clipboard\browser\terminalClipboard.ts" itself,

this used to handle only ctrl+v action on pwsh, if the user tries to paste using right mouse click the multi line paste didn't used to work properly (Not sure if this is a bug or an expected behavior),

Now with the changes I've made the right mouse click also works as expected

It's just that we get the multi line paste alert on ctrl+v as well

![image](https://github.com/user-attachments/assets/7acfd95c-055a-466f-8f9d-8bc38180cb81)